tools/dynamodbConnector.py

```diff
@@ -0,0 +1,103 @@
+import boto3
+import logging
+from botocore.exceptions import ClientError
+
+
+class DynamoDBConnector:
+    """Connector to make the web scraper compatible with AWS Dynamodb. It relies
+    on boto3 and uses the credentials stored in ~/.aws/credentials.
+    """
+
+    def __init__(self):
+        """Initialise the connection, try to create the tables if they don't
+        exist yet. Create a logger instance as well.
```

Comment:
Is that still relevant? I mean we are going to be creating the tables using terraform so if the tables are not there, then it should throw an exception.